lib/main.dart

```diff
@@ -105,8 +112,19 @@ class FlameGame extends Game with KeyboardEvents, TapDetector {
     return distance <= circleRadius;
   }
 
+  bool ballIsOut(){
+      if (ballPosition.y < -300 || ballPosition.y > size.y * 1.5){
```

Comment:
Same thingd here, speak screen position Is better and why under 300 pixels and not 0 ?